pkg/statistics/handle/autoanalyze/priorityqueue/queue_test.go

```diff
@@ -608,3 +609,47 @@ func TestPQCanBeClosedAndReInitialized(t *testing.T) {
 	// Check if the priority queue is initialized.
 	require.True(t, pq.IsInitialized())
 }
+
+func TestPQHandlesTableDeletionGracefully(t *testing.T) {
```

Comment:
Test case for: `[category=stats] [recover="runtime error: invalid memory address or nil pointer dereference"]`